### ISSUE TITLE
LockableModel does not work as proxy

### ISSUE THREAD

**Issue by andreynovikov** [rparent/django-lock-tokens#7]
* django-lock-tokens version: 0.2.1
* Django version: 1.11
* Python version: 3.5
* Operating System: Debian

### Description

I've tried to add locking to Django Admin as described in README, but it fails:

```
File "/usr/local/lib/python3.5/dist-packages/lock_tokens/admin.py" in change_view
  32.             lock_for_session(obj, request.session, force_new=force_new_session_lock)

File "/usr/local/lib/python3.5/dist-packages/lock_tokens/sessions.py" in lock_for_session
  17.     lock_token = LockableModel.lock(obj, token)

File "/usr/local/lib/python3.5/dist-packages/lock_tokens/models.py" in lock
  112.         lock_token = self._lock(token)

Exception Type: AttributeError at /admin/shop/order/95/change/
Exception Value: 'Order' object has no attribute '_lock'
```

I've looked in the code and it looks like proxying is broken. You call `lock` as static method:
https://github.com/rparent/django-lock-tokens/blob/master/lock_tokens/sessions.py#L17
but it's not defined as static:
https://github.com/rparent/django-lock-tokens/blob/master/lock_tokens/models.py#L111

**Comment by rparent** [comment]
You are right, it cannot work as a proxy right now. I will fix that as soon as possible.

I definitely miss some testing for this use case!

**Comment by rparent** [comment]
I just pushed a commit that fixes the bug for python 3 only. Still need to fix for python 2.7.

**Comment by rparent** [comment]
@andreynovikov I just released version 0.2.3 that should fix the problem. If it doesn't, feel free to reopen the issue.

**Comment by andreynovikov** [comment]
Now it works, thanks.
